task1.js

```diff
@@ -0,0 +1,7 @@
+process.stdin.on('data', (data) => {
```

Comment:
JFYI, `'data'` event doesn't guarantee that a chunk of data will be a string ending with a newline symbol. Though, the task statement doesn't explicitly mentions that kind of "strings" should be reversed :)